### ISSUE TITLE
proj_ray_dist_threshold

### ISSUE THREAD

**Issue by nogoing** [POSTECH-CVLab/SCNeRF#16]
Hello. Thank you for great paper and code.

I have one small question.

The threshold value of projected ray distance loss is set to 5, is there a reason why you chose this value?

Also, is this threshold value used even when the camera parameters are initialized with identity matrix and zero vector? (Self-Calibration experiments)
When the camera parameter is coarse or has a bad value, I think the proj_ray_dist loss will be much larger than 5, but wasn't it?
I wonder if this threshold works.

Thank you!

**Comment by jeongyw12382** [comment]
First, the thershold 5 is selected after searching hyperparameters. 

In addition, as mentioned in the paper, we do not use the projected ray distance loss at the very beginning of the training because of the reason you've just mentioned in this issue. I recommend referring to the Curriculum Learning subsection on the main paper. This might help your understanding.

**Comment by jeongyw12382** [comment]
Please let me know if you have any difficulties on using this repository.
You can feel free opening the issue again.
